specs/spec.md

```diff
@@ -0,0 +1,329 @@
+# @TODO
+- [ ] Link to parsers.
+- [ ] attestations
+- [ ] what else needs to be defined in the transport header?
+
+# Hobbits
+
+*Written by Dean Eigenmann <dean@status.im> & Rene Nayman <rene@whiteblock.io>* 
```

Comment:
We can move acknowledgements to the very end of the doc. Let's make sure Matt Elder gets in there too.